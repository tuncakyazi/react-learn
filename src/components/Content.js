import React from 'react';
import PropTypes from 'prop-types';

const Content = props => <div>{props.children}</div>;
/* eslint-disable */
Content.propTypes = {
  children: PropTypes.node.isRequired
};
/* eslint-enable */
export default Content;
