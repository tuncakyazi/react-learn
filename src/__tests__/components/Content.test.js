import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Content from './../../components/Content';

configure({ adapter: new Adapter() });

it('render correctly', () => {
  const snap = shallow(
    <Content>
      <h1>Content</h1>
    </Content>
  );

  expect(snap).toMatchSnapshot();
});
