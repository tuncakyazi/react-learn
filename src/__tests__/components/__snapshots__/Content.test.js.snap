// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Content>
    <h1>
        Content
    </h1>
</Content>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Content
</h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Content",
      },
      "ref": null,
      "rendered": "Content",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Content
</h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Content",
        },
        "ref": null,
        "rendered": "Content",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
